locales/en.json

```diff
@@ -69,6 +69,38 @@
   "You must select an option": "You must select an option",
   "Signature is Required if you Consent": "Signature is Required if you Consent",
   "Social is required": "Social is required",
+  "Medical Report for a Canada Pension Plan Disability Benefit": "Medical Report for a Canada Pension Plan Disability Benefit",
+  "Instructions for the applicant/patient - please read carefully": "Instructions for the applicant/patient - please read carefully",
+  "An application and a medical report are needed by Service Canada to determine if you qualify for a Canada Pension Plan (CPP) disability benefit.": "An application and a medical report are needed by Service Canada to determine if you qualify for a Canada Pension Plan (CPP) disability benefit.",
+  "You (the applicant) must:": "You (the applicant) must:",
+  "complete the Application for a Canada Pension Plan Disability Benefit (ISP1151). The application\ncan be found at www.canada.ca/esdc-forms.": "complete the Application for a Canada Pension Plan Disability Benefit (ISP1151). The application\ncan be found at www.canada.ca/esdc-forms.",
+  "fill out Section 1 and Section 2 of this Medical Report.": "fill out Section 1 and Section 2 of this Medical Report.",
+  "write your Social Insurance Number at the top of each page of this Medical Report.": "write your Social Insurance Number at the top of each page of this Medical Report.",
+  "Your doctor or nurse practitioner must complete Sections 3 to 9 of the Medical Report, sign it, and send it to Service Canada.": "Your doctor or nurse practitioner must complete Sections 3 to 9 of the Medical Report, sign it, and send it to Service Canada.",
+  "DO NOT WAIT for your doctor or nurse practitioner to complete the Medical Report before sending your completed application to Service Canada. The date Service Canada receives your application could affect when your benefit starts.": "DO NOT WAIT for your doctor or nurse practitioner to complete the Medical Report before sending your completed application to Service Canada. The date Service Canada receives your application could affect when your benefit starts.",
+  "Service Canada will help you pay for the cost of the Medical Report by paying up to $85.00 directly to your doctor or nurse practitioner. Any money owing over this amount is your responsibility.": "Service Canada will help you pay for the cost of the Medical Report by paying up to $85.00 directly to your doctor or nurse practitioner. Any money owing over this amount is your responsibility.",
```

Comment:
should these keys be a little more keylike similar to section1.title

Reply:
They should, but they weren't for section 1. I suspect they appear changed due to some re-ordering from the giant #89 merge.

Reply:
Actually, looking at it, there are `start.` translations in the file. I'm just going to remove these lines since they're not used anywhere.

Reply:
Updated to remove the stale keys.